### DISCUSSION TITLE
How to setup AWS authorization to my artillery script

### DISCUSSION THREAD

**Discussion by gabor386** [artilleryio/artillery#1833] [Artillery]
Is there any way to add authorization to my script that my requests are authorized for access AWS Lambda, SQS?
aws signature v4 is need https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html

**Comment by hassy** [comment]
hi @gabor386, have you tried https://github.com/Nordstrom/artillery-plugin-aws-sigv4

**Comment by gabor386** [comment]
@hassy  Yes, and no luck the plugin, i am getting error https://github.com/Nordstrom/artillery-plugin-aws-sigv4/issues/41
seem that they are not supporting/maintaining it anymore.
